Makefile

```diff
@@ -94,3 +94,11 @@ install: clean ## install the package to the active Python's site-packages
 tag: clean
 	git tag -a -m 'version $$(python setup.py --version)' v$$(python setup.py --version)
 	git push --follow-tags
+
+update-deps:
+	pip-compile --upgrade --allow-unsafe requirements/prod.in
+	pip-compile --upgrade --upgrade-package 'typeguard==2.10.0' --allow-unsafe requirements/dev.in
```

Comment:
typeguard==2.10.0 should be in requirements/dev.in, not here

Reply:
looks like we need to hold back `robot` too

Reply:
Ah yes -- both robotframework and robotframework-seleniumlibrary